lib/dataTableHelpers.tsx

```diff
@@ -115,3 +118,43 @@ export function sortByBiospecimenId(rowA: Entity, rowB: Entity) {
 
     return defaultNumericalComparison(rowA, rowB, iteratees);
 }
+
+export function generateColumnsByDataSchema<T>(
+    schemaId: string,
+    schemaDataMap?: { [id: string]: DataSchemaData },
```

Comment:
can you name the `id` parameter more specifically here? this function is pretty abstract so careful naming would be helpful for readability

Reply:
We can name it `dataSchemaId`. We already use these ids in data standard pages. For example:

https://github.com/ncihtan/htan-portal/blob/0608bc381ccbd2723e3b3f550ac68bee74a5f039/pages/standard/biospecimen.tsx#L14-L16 

Actually, we can add an enum like `DataSchemaId` for the ones that we use in the code instead of using hardcoded strings.

Reply:
Added a new enum for clarity